app/Livewire/Analytics.php

```
@@ -0,0 +1,519 @@
+     */
+    protected function appMeta(string $uuid): array
+    {
+        if (isset($this->appMetaCache[$uuid])) {
+            return $this->appMetaCache[$uuid];
+        }
+
+        $app = Application::ownedByCurrentTeam()->with('environment.project')->whereUuid($uuid)->first();
+
+        $domain = null;
+        if ($app) {
+            $first = collect($app->fqdns)->first();
```

PHPStan (2.2.7)

[warning] Unable to resolve the template type TKey in call to function collect
See: https://phpstan.org/blog/solving-phpstan-error-unable-to-resolve-template-type

(argument.templateType)